packages/asset-swapper/package.json

```diff
@@ -53,6 +53,7 @@
         "@0x/orderbook": "^2.2.5",
         "@0x/utils": "^5.4.1",
         "@0x/web3-wrapper": "^7.0.7",
+        "axios": "^0.19.2",
```

Comment:
When I ran `yarn` from the monorepo root locally, I got the root-level `yarn.lock` changes.  Do you have the same on your machine?

```
+++ b/yarn.lock
@@ -3076,6 +3076,12 @@ axios@^0.18.0:
     follow-redirects "^1.3.0"
     is-buffer "^1.1.5"

+axios@^0.19.2:
+  version "0.19.2"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-0.19.2.tgz#3ea36c5d8818d0d5f8a8a97a6d36b86cdc00cb27"
+  dependencies:
+    follow-redirects "1.5.10"
+
 babel-code-frame@^6.22.0, babel-code-frame@^6.26.0:
   version "6.26.0"
   resolved "https://registry.yarnpkg.com/babel-code-frame/-/babel-code-frame-6.26.0.tgz#63fd43f7dc1e3bb7ce35947db8fe369a3f58c74b"
@@ -7659,6 +7665,12 @@ flush-write-stream@^1.0.0, flush-write-stream@^1.0.2:
     inherits "^2.0.1"
     readable-stream "^2.0.4"

+follow-redirects@1.5.10:
+  version "1.5.10"
+  resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.5.10.tgz#7b7a9f9aea2fdff36786a94ff643ed07f4ff5e2a"
+  dependencies:
+    debug "=3.1.0"
+
```

Reply:
Just to close the loop in this context: Steve and spoke elsewhere, and while we don't fully understand the issue (it's perhaps due to Steve's older version of `yarn`), it is not a blocker to him developing locally.